icons/desktop/dock-bottom-16.svg

```diff
@@ -1,4 +1,10 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<svg viewBox="0 0 16 16" width="16" height="16" xmlns="http://www.w3.org/2000/svg" fill="context-fill #0b0b0b"><path d="M12 0a3 3 0 0 1 3 3v10a3 3 0 0 1-3 3H4a3 3 0 0 1-3-3V3a3 3 0 0 1 3-3h8zm0 2H4a1 1 0 0 0-1 1v7h10V3a1 1 0 0 0-1-1zm1 9H3v2a1 1 0 0 0 1 1h8a1 1 0 0 0 1-1z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"
+  fill="context-fill #0b0b0b">
```

Comment:
Fill property should be only `fill="context-fill"`

Reply:
Ok. Is that so the users don't mistakenly assume they're getting the `context-fill` when they're actually getting the fallback?